qcodes/instrument/base.py

```diff
@@ -434,6 +434,15 @@ def snapshot_base(self, update=False):
                 snap[attr] = getattr(self, attr)
         return snap
 
+    def print_readable_snapshot(self, update=False):
```

Comment:
@jenshnielsen I think that would be a good idea. I probably won't have time to do that today. Feel free to add it if you want to make it a quick merge, otherwise I'll probably get to it somewhere in the holidays.